index.html

```diff
@@ -4,7 +4,19 @@
     <meta charset="utf-8">
     <title>js-calc</title>
   </head>
+
   <body>
+
+    <div>
+      <input id="x" type="text"/>
+      <input id="y" type="text"/>
+      <input type="button" onclick ="calculator.sum()" value="sum"/>
```

Comment:
Cool that you added a visual interface, but this will error out because there is no global variable called `calculator` in your code.